Clarify use of prefers-reduced-motion: reduce in CSS example

farizadam opened the pull request:
<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description

This PR updates the @media rule from prefers-reduced-motion to prefers-reduced-motion: reduce in the CSS example to explicitly match the widely used syntax and align with best practices.

### Motivation

The update addresses potential confusion for developers. Although @media (prefers-reduced-motion) works as a boolean, using the explicit : reduce value improves clarity and aligns with other documentation like [web.dev](https://web.dev/articles/prefers-reduced-motion).

### Additional details


### Related issues and pull requests
Fixes #39543

<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

chrisdavidmills reviewed (changes requested):
Hi @farizadam, and thank you for your contribution to MDN! I think the change you have made here is good, however, I wonder if you should go slightly further? The syntax example at the top of the page still shows

```
@media (prefers-reduced-motion) {
```

Maybe you could also change that to be consistent? 

Then, I noticed the place where it says "This keyword value evaluates as true." That makes sense to more experienced developers, but I thought less experienced developers might benefit from a more explicit description, along the lines of "The `reduce` keyword value evaluates as true, therefore `@media (prefers-reduced-motion)` is equivalent to `@media (prefers-reduced-motion: reduce)`."

What do you think?

farizadam commented:
@chrisdavidmills I've updated the PR to address both of your suggestions:

1. Changed the syntax example at the top to use `prefers-reduced-motion: reduce`
2. Added the explicit explanation about the equivalence between `@media (prefers-reduced-motion)` and `@media (prefers-reduced-motion: reduce)`

Thank you for your review feedback!

chrisdavidmills reviewed (approved):
Perfect, nice one @farizadam! I just made a minor formatting fix, but this looks ready to merge now.

Thanks again for your contribution!
